### DISCUSSION TITLE
第3节课笔记(3班)

### DISCUSSION THREAD

**Discussion by matrixgame2018** [InternLM/Tutorial#187] [笔记]
第3节课笔记(3班)

**Comment by hydexi0122** [comment]
https://zhuanlan.zhihu.com/p/676751587

**Comment by 443752009** [comment]
http://t.csdnimg.cn/r5SoN

**Comment by SchweitzerGAO** [comment]
https://schweitzergao.github.io/2024/01/08/mmtutorial-3-note/

**Comment by zzmbhm** [comment]
https://docs.qq.com/doc/DZmpHTVNubU9XeHBr

**Comment by GoldWaterFall** [comment]
https://www.yuque.com/woshijinshui/wwi5m1/fw1qapca13gx5uso?singleDoc# 《基于 InternLM 和 LangChain 搭建你的知识库笔记》

**Comment by GoooHi** [comment]
https://zhuanlan.zhihu.com/p/677421011

**Comment by Tallisgo** [comment]
http://t.csdnimg.cn/dArXU

**Comment by xiexiaoshinick** [comment]
[[笔记和作业]基于 InternLM 和 LangChain 搭建你的知识库 - 知乎 (zhihu.com)](https://zhuanlan.zhihu.com/p/677524722)

**Comment by semple030228** [comment]
https://blog.csdn.net/m0_49289284/article/details/135561304

**Comment by wm901115nwpu** [comment]
https://zhuanlan.zhihu.com/p/676750545

**Comment by G-Mie** [comment]
https://blog.csdn.net/YYYYbhjbhkv_/article/details/135578603

**Comment by semple030228** [comment]
https://blog.csdn.net/m0_49289284/article/details/135561304

**Comment by Gty-2002** [comment]
https://npumgk6rgc.feishu.cn/docx/L9W5dozOdorq7rx0l2kcvQ1WnEc?from=from_copylink 笔记及作业

**Comment by Sampson425** [comment]
https://blog.csdn.net/m0_49908454/article/details/135581616
